caikit_nlp/modules/text_generation/peft_tgis_remote.py

```diff
@@ -202,6 +202,10 @@ def run(
         stop_sequences: Optional[List[str]] = None,
         seed: Optional[np.uint64] = None,
         preserve_input_text: bool = False,
+        input_tokens: bool = False,
+        generated_tokens: bool = True,
+        token_logprobs: bool = True,
+        token_ranks: bool = True,
```

Comment:
Was there a particular reason to set all of these to `True`? These all seem like extra response options that could be helpful for a user but might be extra overhead on computation

Reply:
The main reason being the hard-coded values in `caikit_nlp/toolkit/text_generation/tgis_utils.py` were the very same. So I set the default here as was the current implementation.

Reply:
ah I see, thanks for the reminder. That's likely fine for parity then and user(s) can explicitly send `False` if needed